source/SH3/system/glprogram.cpp

```diff
@@ -51,21 +53,24 @@ GLint sh3_glprogram::Load(const std::string& name)
 
 void sh3_glprogram::Bind()
 {
-
+    if(programID != 0)
+        glUseProgram(programID);
 }
 
 void sh3_glprogram::Unbind()
 {
-
+    glUseProgram(0); // The 'correct' way to unbind a shader, but apparently undefined?!?!
```

Comment:
I don't have a huge issue with this, but as a fix we could use a dummy shader (and have a static class variable refer to it) that is set on `Unbind()`.
The dummy shader would either just render a solid color or an error message if we want to be fancy.